test/MUXControls.Test/MUXControls.Test.Shared.projitems

```diff
@@ -26,6 +26,7 @@
     <LangVersion>$(CSLangVersion)</LangVersion>
   </PropertyGroup>
   <PropertyGroup Condition="$(BuildingWithBuildExe) != 'true'">
+    <UniversalCRTSdkDir Condition="'$(UniversalCRTSdkDir)' == ''">$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Kits\Installed Roots@KitsRoot10)</UniversalCRTSdkDir>
     <UniversalCRTSdkDir Condition="'$(UniversalCRTSdkDir)' == ''">$(MSBuildProgramFiles32)\Windows Kits\10</UniversalCRTSdkDir>
     <WindowsMetadataPath>$(UniversalCRTSdkDir)\UnionMetadata\$(TargetPlatformVersion)</WindowsMetadataPath>
```

Comment:
While you're here, we might as well move the first backslash from this line up to the end of the previous line, as you called out in the PR description.  I don't think that double-backslashes are a problem for MSBuild, but it doesn't hurt to be sure.

Reply:
Added commit addressing this.